FlowCryptUI/Nodes/AttachmentNode.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  AttachmentNode.swift
+//  FlowCryptUI
+//
+//  Created by QSD BiH on 16. 4. 2021..
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import AsyncDisplayKit
+
+public struct Attachment {
+    var name, size: NSAttributedString
+}
+
+public final class AttachmentsNode: CellNode {
+    public struct Input {
+        let name: String
+        let size: String
+
+        public init(
+            name: String,
+            size: String
+        ) {
+            self.name = name
+            self.size = size
+        }
+    }
+
+    private var attachmentNodes: [AttachmentNode] = []
+
+    public init(input: [Input]) {
+        super.init()
+        input.forEach { input in
+            attachmentNodes.append(
+                AttachmentNode(
+                    input: AttachmentNode.Input(name: input.name, size: input.size)
+                )
+            )
+        }
+    }
+
+    public override func layoutSpecThatFits(_: ASSizeRange) -> ASLayoutSpec {
+        return ASInsetLayoutSpec(
+            insets: UIEdgeInsets(top: 8, left: 0, bottom: 8, right: 0),
```

Comment:
should right inset be 0? 
can you please add a screenshot of how this should look like

Reply:
@Kharchevskyi Here's what it looks like on Android:
<img width="380" alt="Screen Shot 2021-04-30 at 10 28 16" src="https://user-images.githubusercontent.com/57806041/116669372-c9c52700-a99e-11eb-8656-ab5204e3129b.png">

And currently on iOS:
![Screen Shot 2021-04-30 at 10 33 58](https://user-images.githubusercontent.com/57806041/116670071-96cf6300-a99f-11eb-92b8-f33f0861d9be.png)



I tried to match this as closely as  I could. 

Also, paperclip image is just a sample since I don't have the right ones. @tomholub Can you send them?

Reply:
We don't need to match THAT closely between Android and iOS. It should be roughly aligned in the sense that there are attachments on the bottom, etc, but exact icons and exact borders etc should rather follow platform habits. For example for iOS download button, I'd use some icon from iOS palette that is available for that purpose, rather than Android icon. (sorry for not clarifying earlier)

Reply:
In other words, when I open the android app, it should look like and android app. When I open iOS app, it should look like an iOS app. When I put the two next to each other, they look similar and clearly from the same company, but each their own "flavor" following the platform.

Reply:
Got it, thanks

Reply:
Great then 🔥
@qsoftdevelopment can you please make border width to be 1 point(0.5 will highly drop performance on drawing). also please make left and right offset to be 8 points and please make round 8 corner radius to follow App Designn Guidelines. And also use more iOS icons